app/components/template-viewer/template-viewer.sass

```diff
@@ -10,3 +10,6 @@
     font-family: $font-family-sans-serif
     margin-bottom: $sbh-margin-bottom
 
+.template-item
+  &:hover
+    background-color: #FFE6B8 
```

Comment:
If we could use a global sass here, that would be preferable. feel free to add to the viz-visualization.sass in boilerplate